front/lib/api/datasets.ts

```diff
@@ -78,10 +78,6 @@ export async function getDatasetHash(
     },
   });
 
-  if (!dataset) {
```

Comment:
Here we need to export the dataset even if it's not in front anymore. If it's not in front just skip description and schema.